src/main/java/org/wmn4j/io/musicxml/MusicXmlReader.java

```diff
@@ -16,7 +16,7 @@
 	/**
 	 * The path to the local MusicXML schema.
 	 */
-	Path SCHEMA = Paths.get("src/wmnlibio/musicxml/musicxml.xsd");
+	Path SCHEMA = Paths.get("src/main/java/org/wmn4j/io/musicxml/schema/musicxml.xsd");
```

Comment:
I think that this Path could be removed from the interface entirely, and the schema file could be handled as a resource by putting it in `src/main/resources/org/wmn4j/io/musicxml`. See https://docs.gradle.org/current/userguide/organizing_gradle_projects.html for more on handling resources in Gradle projects.

Reply:
Fixed :+1: